packages/nextjs/components/scaffold-eth/Contract/utilsContract.tsx

```diff
@@ -160,6 +160,24 @@ const getParsedEthersError = (e: any): string => {
 
   return message;
 };
+const extractKeyValueErrorPairs = (errorMessage: any) => {
```

Comment:
I think the naming of this function could better reflect its purpose, e.g. `extractErrorMessage` or something like that

Reply:
Greate